modules/ols-creating-lightspeed-custom-resource-file-using-cli.adoc

```diff
@@ -53,19 +53,20 @@ spec:
   llm:
     providers:
     - credentialsSecretRef:
-        name: openai-api-keys
+        name: rhelai-api-keys <1>
```

Comment:
Zero domain knowledge here--is this really a user-replaceable value? If so, use angled brackets and underscores. Same elsewhere.

Reply:
done